sec_certs/sample/common_criteria.py

```diff
@@ -241,6 +251,8 @@ class CCHeuristics(Heuristics, ComplexSerializableType):
         cert_id: Optional[str] = field(default=None)
         st_references: References = field(default_factory=References)
         report_references: References = field(default_factory=References)
+        direct_vulnerabilities: Optional[Set[str]] = field(default=None)
```

Comment:
**Thought:** Technically, this could (and should) replace `related_cves` attribute, because it holds exactly this information. I think that you are only expanding to "indirect" vulnerabilities, right?

Reply:
Oh, I see. It does not! Because we are using 3 different states of vulnerabilities, right?
1. Certificate suffers from vuln.
2. Certificate *directly* references some other cert that suffers from vuln.
3. Certificate *indirectly* references some other cert that suffers from vuln.

Can we think of some better name please, so that 2 does not resemble 1, please?